latest/index.bs

```diff
@@ -425,17 +433,30 @@ custom attributes of the plate group under the `plate` key.
   <dt><strong>rows</strong></dt>
   <dd>A list of JSON objects defining the rows of the plate. Each row object
       defines the properties of the row at the index of the object in the
-      list. If not empty, it MUST contain a `name` key specifying the row
-      name.</dd>
+      list. Each row in the physical plate MUST be defined, even if no wells
+      in the row are defined. Each defined row MUST contain a `name` key
+      specifying the row name. The `name` MUST contain only alphanumeric
+      characters, MUST be case-sensitive, and MUST NOT be a duplicate
+      of any other `name` in the `rows` list. Care SHOULD be taken to avoid
+      collisions on case-insensitive filesystems (e.g. avoid using both `Aa`
+      and `aA`).</dd>
   <dt><strong>version</strong></dt>
   <dd>A string defining the version of the specification.</dd>
   <dt><strong>wells</strong></dt>
   <dd>A list of JSON objects defining the wells of the plate. Each well object
-      MUST contain a `path` key identifying the path to the well subgroup.</dd>
+      MUST contain a `path` key identifying the path to the well subgroup.
+      The `path` MUST consist of a `name` in the `rows` list, a file separator (`/`),
+      and a `name` from the `columns` list, in that order. The `path` MUST NOT contain
+      additional leading or trailing directories.
```

Comment:
It just occurs to me that this reads as if the  names for the well row Zarr group and the well Zarr group must be defined in the rows and columns lists. 

My understanding is that the intent was not to enforce this requirement and this was one of the basis for making column/row indices mandatory.

Reply:
Currently `ome-zarr-py` doesn't even read the `path` value for each Well, but assumes the path is `row/column` for every Well:

https://github.com/ome/ome-zarr-py/blob/13b84281732a06aa714648a86d070c6d9e7a40c9/ome_zarr/reader.py#L537

It's assumed the plate is not sparse, but we read the tiles in a try/catch that handles missing Wells.

Reply:
Probably worth a short discussion. Row and column names are already explicitly required here (added in https://github.com/ome/ngff/pull/24/commits/5a1ddc7d57d3c7f96e1d84efdb7a86e701666de6). I don't know how we could allow the names to be optional, but still enforce ```row/column``` as the well path.

Reply:
In vizarr I do the opposite: start with `path` and work out the row and column from that:
https://github.com/hms-dbmi/vizarr/blob/3f2149727923eb38340d8f6590b20ccf93eebc9f/src/ome.ts#L169

In both cases, I am managing without the new `rowIndex` and `columnIndex` because of this rule (which wasn't in the spec before)? With this PR, you could imagine that the `path` could be anything (if you look it up for every Well), allowing you to store the Well images in a flexibly way, using whatever `path` you want.

I'm not too sure of why we need `rowIndex` and `columnIndex` to be defined (instead of calculating them), but with them, I think a file would be invalid if the `path` wasn't equal to `rows[rowIndex]/columns[colIndex]` because you wouldn't know whether to use that or the provided `path`?

Reply:
It looks like I misremembered the set of decisions made in https://github.com/ome/ngff/pull/24#issuecomment-990082917. https://github.com/ome/ome-zarr-py/pull/159 was adjusted to match the specification as defined and additional checks are added to validate the content of well paths against `rows`, `rowIndex`, `columns`, `columnIndex`